pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildPythonPackage, lib, fetchFromGitHub, libcint, libxc, xcfun, blas,
+  numpy, scipy, h5py, python3
+} :
+
+buildPythonPackage rec {
+  pname = "pyscf";
+  version = "1.7.6.post1";
+
+  src = fetchFromGitHub {
+    owner = "pyscf";
+    repo = pname;
+    rev = "f6c9c6654dd9609c5e467a1edd5c2c076f793acc";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
See above. 1.7.6 does not build correctly

Reply:
https://github.com/pyscf/pyscf/issues/872

Reply:
Sounds like it boils down to a single issue. Is it also a single (or a few) commits that fix the issue? If yes, you could simply pull them in as patches.

Reply:
It is just 2-3 commits behind the release tag. Pulling these two as patches would be exactly equivalent to this commit. So I guess the current version is easier?

Reply:
Using version makes automated updates easier and I do not need to verify that not a commit from a fork is fetched.

Reply:
@SuperSandro2000 So I should use the release tag and fetch the missing commits as patches as @markuskowa suggested? The 1.7.6 release tag alone will not work.

Reply:
And as written above 1.7.6.post1 is exactly the version tag that is on PyPi.